app/views/training/pages/certificate.html.slim

```diff
@@ -16,44 +16,49 @@
         =< govuk_link_to 'My account', user_path
         | .
 
-h3.govuk-heading-s Certificate of achievement
```

Comment:
Change this to add some custom classes that will help us target the certificate text more effectively.
i.e. add "cert-14 cert-bold"